WebDriverAgentLib/Utilities/FBKeyboard.m

```diff
@@ -28,10 +29,14 @@ + (BOOL)typeText:(NSString *)text error:(NSError **)error
   if (![FBKeyboard waitUntilVisibleWithError:error]) {
     return NO;
   }
+
+  NSUInteger maxTypingFrequency = [FBConfiguration maxTypingFrequency];
+  [FBLogger logFmt:@"Typing with maximum frequency %lu", maxTypingFrequency];
```

Comment:
Can you cast `maxTypingFrequency` to `(unsigned long)`? Otherwise I cannot land it.